### PR TITLE
Pending try

### PR DESCRIPTION
**Pull request by parkkoon**
Add chainIndex & console message.
Change gameOccupied SLOT_TOPICS_ENCODED
Add global variable for extension
Change to sha index correctly

### CODE REVIEW

File: app/javascripts/helpers/web3Service.jsx [hunk 1]
```diff
@@ -17,9 +30,9 @@ class Web3Service {
     this.slotManagerContract = null;
     this.slotStorageContract = null;
     this.storageAddr = null;
-    this.myOccupiedGameInitWatchers = {};
+    this.myInitializedGameInitWatchers = {};
 
-    if (typeof web3 !== 'undefined') {
+    if (typeof web3 === 'undefined') {
```
Comment (c0smic0wl) [inline app/javascripts/helpers/web3Service.jsx]: it causes error when user doesn't have metamask plugin. you should fix it to "!==".

Comment (parkkoon) [reply app/javascripts/helpers/web3Service.jsx]: I agree.

File: app/javascripts/components/playSlot/actions.jsx [hunk 2]
```diff
@@ -162,30 +162,35 @@ export function requestToPlayGame(playInfo, stopSpinFunc) {
...
-        },
-      });
-    } catch (err) {
-      Toast.notie.alert({
-        type: 'error',
-        text: 'There was an error for playing a slot machine',
-        stay: true,
+    const rewardPromise = Web3Service.getSlotResult(playInfo.slotMachineContract);
+    const playPromise = Web3Service.playSlotMachine(playInfo);
+    await Promise.all([rewardPromise, playPromise])
```
Comment (fish-sushi) [inline app/javascripts/components/playSlot/actions.jsx:167]: I think rewardPromise should be executed after playPromise being done

Comment (parkkoon) [reply app/javascripts/components/playSlot/actions.jsx:167]: No, play contract can be decided with reward event. So those promises have to be synchronously executed.

Comment (fish-sushi) [reply app/javascripts/components/playSlot/actions.jsx:167]: I agree.

**Pull request merged**